p2p/transport/quic/transport.go

```diff
@@ -269,7 +269,7 @@ loop:
 }
 
 // Don't use mafmt.QUIC as we don't want to dial DNS addresses. Just /ip{4,6}/udp/quic
```

Comment:
Does this comment need updating? `/ip{4,6}/udp/quic-v1`

Reply:
Yes. Done.